src/cpp/src/visual_language/input_embedder.cpp

```diff
@@ -764,11 +764,6 @@ class InputsEmbedderLLaVANext : public InputsEmbedderLLaVA {
         size_t patch_seq_len = shape[1];
         size_t embed_dim = shape[2];
 
-        OPENVINO_ASSERT(
-            num_patches == num_patch_height * num_patch_width,
-            "Number of patches does not match the specified grid size"
-        );
-
```

Comment:
Why this assert is removed?

Reply:
It was triggered and I was hoping that's something that isn't needed :)